adagrams/game.py

```diff
@@ -1,13 +1,129 @@
 from random import randint
 
-def draw_letters():
-    pass
+def draw_letters(): # No parameters
+    
+    # This letter pool should reflect the distribution of letters as described in the Readme file Wave 1
+    distribution_of_letters = { "A": 9, "B": 2, "C": 2, "D": 4, "E": 12, "F": 2, "G": 3, "H": 2, "I": 9, "J": 1, "K": 1, "L": 4, "M": 2, "N": 6, "O": 8, "P": 2, "Q": 1, "R": 6, "S": 4, "T": 6, "U": 4, "V": 2, "W": 2, "X": 1, "Y": 2, "Z": 1 }
+    
+    letter_pool = [] # This represents the pool of letters that the player can draw from
```

Comment:
```suggestion
    letter_pool = []
```

The variable name `letter_pool` is descriptive enough for me to understand what this variable represents. Prefer to remove comment that doesn't add additional details about how this function works.